app/src/main/java/com/android/sample/ui/navigation/BottomNavigationMenu.kt

```diff
@@ -0,0 +1,31 @@
+package com.android.sample.ui.navigation
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+
+@Composable
```

Comment:
Maybe add a composable representing the standard navigation bar that we will use the most, and make use of default parameter to avoid boilerplate code. Would need to pass navigationAction as param though